css/selectors/descendant.json

```diff
@@ -9,12 +9,28 @@
             "webview_android": {
               "version_added": true
             },
-            "chrome": {
-              "version_added": "1"
-            },
-            "chrome_android": {
-              "version_added": true
-            },
+            "chrome": [
+              {
+                "version_added": true
```

Comment:
This should go after the `"version_added": "61"` entry, and have `"version_removed": "61"` added to it.